statistics/handle/update.go

```diff
@@ -499,6 +499,18 @@ func (h *Handle) dumpTableStatCountToKV(id int64, delta variable.TableDelta) (up
 	return
 }
 
+// UpdatePredicateStatsMeta update the predicate stats meta for this Table.
+func (h *Handle) UpdatePredicateStatsMeta(
```

Comment:
Do we have some existing cache mechanisms to cache the delta infos, and do the real update period by period? I'm a little worried about the overhead here. I guess the read operations might be executed mutiple times during optimization, and might bring heavy overhead

Reply:
This PR is implemented without caching. I have instructions in the group before, and now I have synchronized to the PR's title.

Reply:
So, there will be another pr that will implement cache to make update history stats less impact, right?

Reply:
Yes.